python/tests/test_highlevel.py

```diff
@@ -3867,6 +3867,21 @@ def test_branch_length_empty_tree(self):
         assert tree.branch_length(1) == 0
         assert tree.total_branch_length == 0
 
+    @pytest.mark.parametrize("root_threshold", [1, 2, 3])
```

Comment:
What happens with pathological values of `root_theshold`? (-ve, fractional, infinities) Always good to test.

Reply:
Good point. This is actually tested elsewhere, but not thoroughly enough, it appears. See #2674.

I'll add an extra test in when that PR is merged.

Reply:
The root_threshold stuff is now tested elsewhere in https://github.com/tskit-dev/tskit/pull/2674 so I think we're covered now